tests/unit/jobserver/management/commands/test_release.py

```diff
@@ -0,0 +1,147 @@
+import csv
+from io import StringIO
+
+import pytest
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management import call_command
+
+from jobserver.models import Workspace
+
+from .....factories import (
```

Comment:
Elsewhere I started moving to `tests.factories` after ~3 dots as it was getting a bit out of hand!

Reply:
It did feel like a lot of dots.....